test/views/declaration/LocationViewSpec.scala

```diff
@@ -302,8 +329,8 @@ class LocationViewSpec extends UnitViewSpec with ExportsTestData with Stubs with
             Some(TestHelper.createRandomAlphanumericString(36)),
             Some(TestHelper.createRandomAlphanumericString(4)),
             Some(TestHelper.createRandomAlphanumericString(71)),
-            Some(TestHelper.createRandomAlphanumericString(10)),
             Some(TestHelper.createRandomAlphanumericString(36)),
+            Some(TestHelper.createRandomAlphanumericString(10)),
```

Comment:
Here you also have the same pattern as in the cases above. In my opinion it would be better to use `bind` method and field names.
Actually, here you should not switch the lines, as you have not changed the order in the constructor.

Reply:
Fixed